analyses/tmb-compare-tcga/TMB_d3b_code/code/02_cumulative_freq_TMBplot.py

```diff
@@ -41,15 +41,30 @@
 tmbfile = pd.read_csv(args.tmb_scores, sep="\t")
 
 # Choosing disease list to plot
-#tmbfile["new_TMB"] = tmbfile["TMB"]+1
-value_to_add = np.min(tmbfile[tmbfile["TMB"] != 0]["TMB"])*0.8
-tmbfile["new_TMB"] = tmbfile["TMB"].replace({0: value_to_add})
-tmbfile = tmbfile.sort_values(by=['new_TMB'])
+#tmbfile["new_TMB"] = ( tmbfile["count"] + 1 * 1000000) / tmbfile["bedlength"]
+#tmbfile["new_TMB"] = tmbfile["count"] + 1 / tmbfile["bedlength"]
 
 
-disease_number = tmbfile.groupby("cohort").count()["new_TMB"]
+tmbfile["new_TMB"] = tmbfile.apply(lambda x:
+    (x["count"] + 1 * 1000000) / x["bedlength"] if x["count"]==0
+    else x["TMB"],axis=1)
```

Comment:
For consistency of comparison, this should be applied to all TMB. 
```suggestion
    (x["count"] + 1 * 1000000) / x["bedlength"],
    axis=1)
```

Reply:
That's fine, but we should add another parentheses around count + 1

`tmbfile["new_TMB"] = tmbfile.apply(lambda x:
    ((x["count"] + 1) * 1000000) / x["bedlength"],
    axis=1)`

I updated the code in the commit below